### PR TITLE
fix(ui): Fix download to CSV flow using Scroll across entities api

### PR DESCRIPTION
**Pull request by jjoyce0510**
## Summary

In this PR we move to using scrollAcrossEntities api which allows for deep pagination against elasticsearch. This fixes download as csv which did not previously work for large data sets (greater than a few thousand)

Also, making scroll correctly apply the selected View. 

## Validation

Tested against 10k entities locally.

## Status

Ready for review

## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-web-react/src/graphql/scroll.graphql [hunk 1]
```diff
@@ -0,0 +1,399 @@
+fragment downloadSearchResults on Entity {
```
Comment (aditya-radhakrishnan) [inline datahub-web-react/src/graphql/scroll.graphql:1]: Could we instead add another subdirectory called scroll? Imo this makes more sense to be in a path like `scroll/search.graphql`

File: datahub-web-react/src/app/search/utils/constants.ts [hunk 2]
```diff
@@ -94,3 +94,5 @@ export enum UnionType {
     AND,
     OR,
 }
+
+export const SCROLL_KEEP_ALIVE_TIME = '10m'; // TODO: Determine if this is sufficient.
```
Comment (aditya-radhakrishnan) [inline datahub-web-react/src/app/search/utils/constants.ts:98]: Can we make this an environment variable instead?

**Review by aditya-radhakrishnan — COMMENTED** [review]
Generally looks good, have some small comments. Think we should use environment variables when possible to allow customization when deploying.

Can we also add some Cypress tests to confirm this is working properly for large numbers of search results?

File: datahub-web-react/src/app/search/utils/constants.ts (hunk 2 above)
Comment (jjoyce0510) [reply datahub-web-react/src/app/search/utils/constants.ts:98]: Environment variable? That means extending a config endpoint right?

File: datahub-web-react/src/graphql/scroll.graphql (hunk 1 above)
Comment (jjoyce0510) [reply datahub-web-react/src/graphql/scroll.graphql:1]: Why is that? This directory is pretty flat already and i am not sure that this warrants a change

**Comment by jjoyce0510** [comment]
Adding Cypress Tests for large downloads is kinda a scary proposition because it can take a long time to get the requisite amount of data into the instance. Took me ~30 minutes to get ~7k entities into my own local instance. I'd hesitate to do that. Maybe a better solve is to have a download as csv cypress test generally?

File: datahub-web-react/src/app/search/utils/constants.ts (hunk 2 above)
Comment (jjoyce0510) [reply datahub-web-react/src/app/search/utils/constants.ts:98]: Kinda blows up scope to adding this to our appConfig GraphQL API. Let me confirm with Ryan on platform team whether this is something we'd need to allow config on

**Comment by jjoyce0510** [comment]
Just chatted with Ryan - he mentioned that we can omit the parameter altogether for now! Going to remove it...

File: datahub-web-react/src/graphql/scroll.graphql (hunk 1 above)
Comment (jjoyce0510) [reply datahub-web-react/src/graphql/scroll.graphql:1]: I was actually considering putting this in either a) search.graphql or in a new b) download_csv.graphql

**Review by aditya-radhakrishnan — APPROVED** [review]

**Pull request merged**